app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -125,7 +125,7 @@
         \n-&#032;应用名称的一部分（比如，Blue Line Console）
         \n-&#032;包名的一部分（比如，net.nhiroki.bluelineconsole）
         \n-&#032;URL（比如，https://example.com/）
-        \n-&#032;算数计算（比如， 2+3*5, 1inch in cm, 1m+1inch, 1m+1inch in cm）
+        \n-&#032;计算公式 （比如， 2+3*5, 1 英寸等于多少厘米, 1 米+1 英寸, 1 米加 1 英寸等于多少厘米）
```

Comment:
" 1inch in cm", "1m+1inch", and "1m+1inch in cm" cannot be translated because this is what users must type exactly (Chinese or Japanese input is not accepted.).

Reply:
Change reverted.